readme.md

```diff
@@ -0,0 +1,12 @@
+# React Uploader
+react component that depends on UploadCare to persist uploads.
+
+### Develop
+* `npm run dev` starts webpack-dev-server
+
+### Distribute
+* `npm run build` write to dist folder
+* `npm version patch` updates version in package.json
+* `npm publish` publishes package to public registry
+* `npm install react-uploader` from another project to install
+* See src/index.html for how to initialize ReactUplaoder
```

Comment:
Yea in here lets add a seciton about how you initialize everything on the dom, doesnt have to be super detailed but i know thats been confusing